denoise/README.md

```diff
@@ -0,0 +1,32 @@
+# Denoising Autoencoder
+
+#### A Denoising Autoencoder is an autoencoder which learns to reproduce the input from a compressed representation of the output. The main difference between vanilla and denoisisng autoencoder is that a denoising autoencoder is trained on a noisy original input. This noise can be of any type(for eg: Gaussian Noise). The encoder neural network is trained to ignore the noise and create an efficient representation while the decoder tries to reconstruct the output.
```

Comment:
Denoising AE isn't used for reproducing the input.

Reply:
Rewrote the whole definition again.